src/test/java/resources/authentication.feature

```diff
@@ -0,0 +1,29 @@
+@gov
+Feature: Authentication test for services routed through kong & keycloak
+
+  @keycloak-user-setup @keycloak-user-teardown @kong-teardown
+  Scenario: Unauthorized channel-connector test
+    Given I have tenant as "gorilla"
+    And I can mock TransactionChannelRequestDTO
+    And I register service with url "https://ph-ee-connector-channel.paymenthub.80.svc" and "https" protocol
```

Comment:
move these values to the application yaml as these values are tightly coupled to the current installation